test/test_ops.py

```diff
@@ -542,7 +542,7 @@ def test_errors(self, device, op):
     @skipMeta
     @onlyNativeDeviceTypes
     @ops([op for op in op_db if op.error_inputs_sparse_func is not None], dtypes=OpDTypes.none)
-    @parametrize("layout", (torch.sparse_csr, torch.sparse_csc, torch.sparse_bsr, torch.sparse_bsc, torch.sparse_coo))
+    @parametrize("layout", (torch.sparse_bsr, torch.sparse_bsc, torch.sparse_coo))
```

Comment:
This change is not needed as it will disable existing tests for CSR and CSC samples.